src/App.jsx

```diff
@@ -15,9 +19,14 @@ function App() {
         <Routes>
           <Route path="/" element={<Home />} />
           <Route path="/articles/:article_id" element={<IndividualArticle />} />
+          <Route
+            path="/articles/:article_id/comments"
+            element={<ArticleComments />}
+          />
```

Comment:
There's no need for this path as you have rendered your comments underneath the individual articles 😄